.github/workflows/pr.yml

```diff
@@ -17,6 +17,8 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
+          ref: ${{ github.event.pull_request.head_sha }}
+
```

Comment:
For some reason the CI fails without this. From what I'm reading my change makes it run against the PR's current commit, while it previously ran against an auto-generated commit that would be present without merging.

I'm not sure if this would cause any issues, but it's the only way I've gotten the CI to pass, and I was fine with it and then debugging everything in the future if issues came up.